app/facades/revenue_facade.rb

```diff
@@ -0,0 +1,7 @@
+class RevenueFacade
+  def self.merchant_revenue(merchant)
+    result = merchant.invoices.joins(:invoice_items).joins(:payments).select("sum(invoice_items.quantity * invoice_items.unit_price) as revenue").where("payments.result = 'success' AND status = 'shipped'").group(:id).order("revenue desc").first.revenue
+
+    Revenue.new(result)
```

Comment:
This is creating a new instance of a revenue object for serialization standardization purposes.